### PR TITLE
MINOR: Resolve merge conflict

### PR DESCRIPTION
**Pull request by vvcephei**
Follow-on to #595

There were two commits that caused conflicts.
This merges them both and one innocent commit in the middle.
This commit also resolves the conflicts.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: gradle/dependencies.gradle [hunk 1]
```diff
@@ -62,14 +62,14 @@ versions += [
...
-  gradle: "7.0.2",
+  gradle: "7.1.1",
   grgit: "4.1.0",
   httpclient: "4.5.13",
   easymock: "4.3",
   jackson: "2.12.3",
   jacoco: "0.8.7",
   javassist: "3.27.0-GA",
-  jetty: "9.4.40.v20210413",
+  jetty: "9.4.42.v20210604",
```
Comment (vvcephei) [inline gradle/dependencies.gradle:72]: This was the conflict that I resolved.

**Review by vvcephei — COMMENTED** [review]
NB: Aside from the line I called out below, the only other merge-conflict resolution I did was to ignore the changes from the AK Jenkinsfile.

**Review by kowshik — APPROVED** [review]

**Comment by vvcephei** [comment]
There's a downstream build failure for SR: https://jenkins.confluent.io/job/Confluentinc%20Contributors/job/kafka/job/PR-597/1/execution/node/162/log/

And another downstream failure for KSQL: https://jenkins.confluent.io/job/Confluentinc%20Contributors/job/kafka/job/PR-597/1/execution/node/166/log/

If possible, we should try to get those projects to fix their usages before merging this PR.

Note, these are the same failures as in #597

**Comment by andrewegel** [comment]
Merged this here: https://github.com/confluentinc/kafka/commit/6dae77ce360eaa29ac46838446affe1c214e663c

**Pull request closed without merging**